src/pages/manage-users/[id].tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import type { InferGetServerSidePropsType, GetServerSideProps } from "next";
+import { getAttendees, getEventById } from "@/lib/firebase/firestore";
+import AttendeeList from "@/components/AttendeeList";
+import { Attendee } from "@/interfaces/UserInterface";
+
+function ManageUsers({
+  attendees,
+  name,
+}: InferGetServerSidePropsType<typeof getServerSideProps>) {
+  return (
+    <div className="min-h-screen bg-gray-100 py-8 px-4">
+      <h1 className="text-4xl font-bold text-center text-indigo-600 mb-8">
+        {name ? name : "There was an error fetching this event"}
+      </h1>
+      {attendees ? (
```

Comment:
If the attendees here is an empty list, the error would not render. Check for the empty case